### ISSUE TITLE
About ResNet18 experiment on CIFAR 10

### ISSUE THREAD

**Issue by ruili-pml** [team-approx-bayes/bayesian-sam#2]
Hi,

Thank you for your excellent work. I have two questions about the ResNet18 experiments on CIFAR10 and would be great if you could answer them.

Question 1: I was trying to reproduce your results in PyTorch and I found that the hyperparameters you provided in paper's appendix were not the same as the ones on github. In paper's appendix ```priorprec``` is set to 10 but in the repo it's set to 40? 

Question 2: I'm not very familiar with Jax so I'm not sure how did you deal with BatchNorm's running statistics during training. As b-SAM have multiple forward pass within one update, I think the running statistics needs to be turned off and turned on. I was wondering have you experimented with when to turn it on and off? 

Thanks,
Rui

**Comment by moellenh** [comment]
Hi, this is probably way outdated so I am closing the issue.  Sorry for not noticing this.  Im answering the questions in case someone else has the same ones. 

**Q1:** Number in paper appendix seems to be a typo unfortunately. 
**Q2:** The BN statistics will get updated twice, once at the noisy point and once at the SAM point -- possibly turning off the updating for one of them could boost the performance.
